### DISCUSSION TITLE
Hyperparameter tuning

### DISCUSSION THREAD

**Discussion by jqloh** [dfki-ric/movement_primitives#66] [Q&A]
Hi, is there a way to systematically tune DMPs to determine the optimal alpha, beta and number of Gaussian kernels implemented yet?

**Comment by AlexanderFabisch** [comment]
Hi @jqloh ,

Alpha and beta don't have to be tuned. The number of rbfs can be tuned manually. If you want to automize this, we would first have to define what we want to optimize. Increasing the number should always reduce the training error.
